src/api/routes/offer.js

```diff
@@ -69,6 +69,7 @@ module.exports = (app) => {
         when(
             (req) => !req.body?.isHidden,
             (req, res, next) => companyMiddleware.verifyMaxConcurrentOffers(req.user?.company || req.body.owner)(req, res, next)),
+        companyMiddleware.profileComplete,
```

Comment:
Move this validator right after the validators.create, so that we can save some work in case companies haven't fully registered yet (it's faster to just do a boolean check every time, instead of running the verifyMaxConcurrentOffers algorithm ever time)